src/foam/nanos/ruler/Rule.js

```diff
@@ -140,6 +140,14 @@
       documentation: 'Defines if the rule needs to be applied before or after operation is completed'+
       'E.g. on dao.put: before object was stored in a dao or after.'
     },
+    {
+      class: 'Boolean',
+      name: 'async',
```

Comment:
Does this cause any issues with the async reserved word in JS? Can you configure this through the GUI? I just did an experiment and it seems fine.